scripts/lib/translate-api.spec.js

```diff
@@ -1,5 +1,7 @@
 var expect = require('chai').expect;
```

Comment:
Since you renamed this file, also need to update the package.json and travis files:
https://github.com/aws/aws-sdk-js/blob/f16d34054deb5dea47e64682272d369832f96553/package.json#L142
(Oops, doesn't look like travis has the current test: https://github.com/aws/aws-sdk-js/blob/f16d34054deb5dea47e64682272d369832f96553/.travis.yml#L31)